corehq/apps/reports/standard/users/reports.py

```diff
@@ -175,7 +178,16 @@ def rows(self):
             self.pagination.start:self.pagination.start + self.pagination.count
         ]
         for record in records:
-            yield self._user_history_row(record, self.domain, self.timezone)
+            yield self._user_history_row(record, self.domain, self.timezone, for_export=False)
+
+    # Override parent method to add new lines to cell values of certain columns
+    @property
+    def export_rows(self):
+        records = self._get_queryset().order_by(self.ordering)[
```

Comment:
Yeah I ran into a bug adding the helper method at first and quit because it seemed to me too like it was on the borderline, but I just tried again and was able to add it (`get_rows`).